closed/autoconf/custom-hook.m4

```diff
@@ -540,3 +547,36 @@ AC_DEFUN([CONFIGURE_OPENSSL],
   AC_SUBST(BUILD_OPENSSL)
   AC_SUBST(OPENSSL_CFLAGS)
 ])
+
+# Create a tool wrapper for use by cmake.
+# Consists of a shell script which wraps commands with an invocation of fixpath.
```

Comment:
Why do we need this? Why can't we simply ask cmake to use `CC=fixpath cl ...`?

Reply:
It appears to work the first time, but when rebuilding,  sometimes it just stops working. I believe it occurs when cmake decides it needs to re-run itself and needs to re-test the compilers.

Reply:
Wouldn't it be better (and perhaps simpler) to just fix that?

Reply:
no. It is a limitiation/bug in cmake itself

Reply:
I'm not convinced: I think we should explore this further (later).